Cabal-syntax/src/Distribution/Fields/Pretty.hs

```diff
@@ -59,7 +59,7 @@ showFields rann = showFields' rann (const id) 4
 -- | 'showFields' with user specified indentation.
 showFields'
   :: (ann -> CommentPosition)
-     -- ^ Convert an annotation to lined to preceed the field or section.
+     -- ^ Convert an annotation to lined to precede the field or section.
```

Comment:
Grammar should be fixed, not just spelling. 

Suggesting 

```suggestion
     -- ^ Convert an annotation to a line preceding the field or section.
```

Reply:
I think it's clear enough to me, but I wouldn't object to the suggestion.

Reply:
It doesn't make sense with just the spelling change?